storage-sqlx/tests/purge.rs

```diff
@@ -0,0 +1,183 @@
+use event_sauce::{
+    prelude::*, AggregateCreate, AggregatePurge, DBEvent, Entity, Event, Persistable,
+    PurgeBuilderExecute, PurgeEntityBuilder,
+};
+use event_sauce_storage_sqlx::{SqlxPgStore, SqlxPgStoreTransaction};
+// use event_sauce::UpdateEntity;
+use sqlx::{postgres::PgQueryAs, PgPool};
+use std::convert::TryFrom;
+use uuid::Uuid;
+
+#[derive(
+    serde_derive::Serialize,
+    serde_derive::Deserialize,
+    sqlx::FromRow,
+    event_sauce_derive::Entity,
+    PartialEq,
+    Debug,
+)]
+#[event_sauce(entity_name = "crud_test_users")]
```

Comment:
This conflicts with another test.

```suggestion
#[event_sauce(entity_name = "crud_test_users_purge")]
```

Reply:
oh, we don't clean the DB between tests do we?

Reply:
I don't think so. IIRC the test harness in this crate is simpler to the cloud project.